code-challenges.test.js

```diff
@@ -26,29 +58,95 @@ var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saf
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
+describe("miniMax", () => {
+    var nums1 = [3, 56, 90, -8, 0, 23, 6]
+    var output1= [-8, 90]
+    var nums2 = [109, 5, 9, 67, 8, 24]
+    var output2= [5, 109]
+    it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order", () => {
+      expect(miniMax(nums1)).toEqual(output1)
+      expect(miniMax(nums2)).toEqual(output2)
+    })
+})
+
+// var nums1 = [3, 56, 90, -8, 0, 23, 6]
 // Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
+// var nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
+// We are going to need a test
+//the function const miniMax (array) =>{}
+    //takes in array of numbers and sorts them.
+    //it will then return a NEW array of the smallest and largest number
+    // maybe a .push into new array first array 
+    //I will need to git find a method that will get the last value in the array.
+    // look into Math.min()/Max
+
+/// below is my refactored function using Math.min()/Max
+// b) Create the function that makes the test pass.
+const miniMax = (array) =>{
+    let newArray = []
+    newArray.push(Math.min(...array))
+    newArray.push(Math.max(...array))
+    return newArray
+}
```

Comment:
Nice job! As a fun refactor you could take your logic and drop it right into the array. `return [Math.min(...array), Math.max(...array)]`